sig-policy/policy-charter.md

```diff
@@ -0,0 +1,35 @@
+# SIG Policy charter
+
+## Scope
+
+SIG Policy covers governance capability, security remediation, and configuration management for the Open Cluster Management (OCM) project.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Review and approve policy related OCM enhancements.
+- APIs used for OCM policies.
```

Comment:
This would need more clarification, e.g. any certain api groups? 

How is API upgrade/deprecation are handled? I assume it will follow the same rule for other APIs.